internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/integration/utils/VeniceTwoLayerMultiRegionMultiClusterWrapper.java

```diff
@@ -481,4 +485,10 @@ public void failPushInRegion(String regions) {
       }
     }
   }
+
+  public Properties getPubSubClientProperties() {
```

Comment:
same here: Can we use a Properties object for pubBrokerDetails to avoid recreating it on every call? Alternatively, we could just return an unmodifiable map.